test-integration/scripts/10-install-jhipster.sh

```diff
@@ -54,10 +54,12 @@ if [[ "$JHI_REPO" == *"/generator-jhipster" || "$JHI_GEN_BRANCH" == "local" ]];
     cd "$JHI_CLI_PACKAGE_PATH"
     git --no-pager log -n 10 --graph --pretty='%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit
     npm ci --prod
+    npm run build
     npm install -g "$JHI_CLI_PACKAGE_PATH"
 
 elif [[ "$JHI_GEN_BRANCH" == "release" ]]; then
     echo "*** generator-jhipster: use release version"
+    npm run build
```

Comment:
```suggestion
```

It’s installing from package, is necessary?

Reply:
Good call. No, I don't think it needs to. I removed it.